src/deserialize/mod.rs

```diff
@@ -34,7 +78,7 @@ fn tokenize(edn: &str) -> Vec<String> {
         .collect()
 }
 
-fn parse<'a>(tokens: &'a [String]) -> Result<(Edn, &'a [String]), Error> {
+pub(crate) fn parse<'a>(tokens: &'a [String]) -> Result<(Edn, &'a [String]), Error> {
```

Comment:
Just a loose comment, I think that `parse_word` should return `Edn::Empty` instead of and Error if the string is empty, what do you think?

Reply:
This is non blocker, maybe a issue

Reply:
Hmmm, maybe that's a good idea actually, I will think about a little than I respond you :slightly_smiling_face:

Reply:
Hmm okay, why do we have `Edn::Empty`? In the spec does it say something about an empty string being valid `edn`?

If there's nothing on it about it, I think we should remove this variant from `Edn` enum.